matador/plotting/pxrd_plotting.py

```diff
@@ -32,13 +32,13 @@ def plot_pxrd(pxrds, two_theta_range=(10, 70), figsize=None):
         pxrds = [pxrds]
     if figsize is None:
         height = max(0.5, 5/len(pxrds))
-        figsize = (8, height)
+        figsize = (10, height)
```

Comment:
Can you leave the default and just pass your own figsize when you call it?

Reply:
defaults for other figs are 10 though but yes can do